cmd/lukso/start.go

```diff
@@ -84,21 +83,28 @@ func (dependency *ClientDependency) Stop() error {
 
 func startClients(ctx *cli.Context) error {
 	log.Info("Looking for client configuration file...")
-	_, err := os.Stat(config.Path)
-	if err != nil {
-		log.Error("Client configuration file not found - please make sure that you downloaded your clients")
+	if !cfg.Exists() {
+		log.Error(folderNotInitialized)
 
 		return nil
 	}
 
-	cfg := config.NewConfig(config.Path)
-	err = cfg.Read()
+	err := cfg.Read()
 	if err != nil {
 		log.Errorf("Couldn't read from config file: %v", err)
 
 		return err
 	}
 
+	// TODO for now just check if installed - when multiple clients will be supported we can run it generically
+	executionClient := cfg.Execution()
+	consensusClient := cfg.Consensus()
+	if executionClient == "" || consensusClient == "" {
+		log.Error("No selected client found in config. Please make sure that you have installed your clients.")
```

Comment:
```suggestion
		log.Error("No selected client found in config file [cli-config.yml - maybe use var here]. Please make sure that you have installed your clients. You can use the install command to install clients.")
```

Reply:
What about `No selected client found in LUKSO configuration file (...)`? To keep it sort of ambiguous from user? We don't want him to tinker with this file anyway, only CLI should be responsible for editing it.

Reply:
ok